packages/marketplace/src/styles/pages/settings.scss

```diff
@@ -0,0 +1,3 @@
+.user-info-section {
+  color: #74818d;
```

Comment:
This color is `$grey` from `packages/marketplace/src/styles/base/colors.scss`, just use it instead